servers/quarkus-common/src/main/java/org/projectnessie/quarkus/providers/storage/JdbcBackendBuilder.java

```diff
@@ -44,11 +44,14 @@ public class JdbcBackendBuilder implements BackendBuilder {
 
   @Override
   public Backend buildBackend() {
-    if (!DatabaseKind.isPostgreSQL(databaseKind) && !DatabaseKind.isH2(databaseKind)) {
+    if (!DatabaseKind.isPostgreSQL(databaseKind)
```

Comment:
Please revert this change; let's not touch the quarkus modules, otherwise this PR will become a huge change.

Reply:
ok i thought this might be needed as well